core/src/tasks/record/export.ts

```diff
@@ -88,10 +74,12 @@ export class RecordExport extends RetryableTask {
       }
```

Comment:
Looks like we're still setting `processedAt` and updating import state here, which would not be necessary

Reply:
Missed that one, thanks! e9ade5adcf213ca02206264c909bc77fdcde1168